bulk_grades/api.py

```diff
@@ -344,6 +359,18 @@ def preprocess_row(self, row):
         self._users_seen.add(row['user_id'])
         return operation
 
+    def read_file(self, thefile):
```

Comment:
Should this be removed ("temporary/do not merge method")

Reply:
Yes. Now that I've merged the super-csv PR, I should be able to update requirements and remove this